tests/test_core.py

```diff
@@ -4174,12 +4174,10 @@ def test_dylink_zlib(self):
     run_process([PYTHON, path_from_root('embuilder.py'), 'build', 'zlib'])
 
     zlib = shared.Cache.get_path(os.path.join('ports-builds', 'zlib', 'libz.a'))
-    side = [zlib]
-    with env_modify({'EMCC_FORCE_STDLIBS': 'libcextra'}):
```

Comment:
do we not need this env change? I don't remember why we ever needed it to be honest..

Reply:
I think its legacy... otherwise the MAIN_MODULE might not include enough of libc for zlib?

Now MAIN_MODULE get the whole of libc I think ???

Reply:
Ah yes, I think you're right, it must have been a workaround for a linking issue like that. Yes, main module will generally get all of libc by default.